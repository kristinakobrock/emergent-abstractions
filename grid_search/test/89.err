  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 16.65it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:02, 20.76it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 22.11it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 21.95it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 20.65it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 20.41it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 21.99it/s] 46%|##########################################3                                                 | 23/50 [00:01<00:01, 21.64it/s] 52%|###############################################8                                            | 26/50 [00:01<00:01, 21.53it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 23.29it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 26.03it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 26.67it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 27.22it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 26.63it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 26.05it/s] 98%|##########################################################################################1 | 49/50 [00:02<00:00, 25.74it/s]100%|############################################################################################| 50/50 [00:02<00:00, 23.90it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.42it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 25.17it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 26.86it/s]100%|############################################################################################| 13/13 [00:00<00:00, 27.89it/s]100%|############################################################################################| 13/13 [00:00<00:00, 27.20it/s]
