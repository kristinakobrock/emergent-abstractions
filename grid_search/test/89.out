# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6803272366523743, "acc": 0.5945839285850525, "length": 3.5693061351776123, "mode": "train", "epoch": 1}
{"loss": 0.6698877811431885, "acc": 0.6214828491210938, "length": 3.87109375, "mode": "test", "epoch": 1}
{"loss": 0.6668119430541992, "acc": 0.6241703033447266, "length": 3.5647382736206055, "mode": "train", "epoch": 2}
{"loss": 0.6668570041656494, "acc": 0.6273499131202698, "length": 3.90625, "mode": "test", "epoch": 2}
{"loss": 0.6651697158813477, "acc": 0.6195467114448547, "length": 3.5613739490509033, "mode": "train", "epoch": 3}
{"loss": 0.6666802167892456, "acc": 0.626953125, "length": 3.73046875, "mode": "test", "epoch": 3}
{"loss": 0.663055956363678, "acc": 0.6254871487617493, "length": 3.5396127700805664, "mode": "train", "epoch": 4}
{"loss": 0.6595840454101562, "acc": 0.6273055672645569, "length": 3.16796875, "mode": "test", "epoch": 4}
{"loss": 0.6620622873306274, "acc": 0.6253367066383362, "length": 3.563466787338257, "mode": "train", "epoch": 5}
{"loss": 0.6575911045074463, "acc": 0.627734363079071, "length": 2.5234375, "mode": "test", "epoch": 5}
{"loss": 0.660619854927063, "acc": 0.6254850029945374, "length": 3.5387046337127686, "mode": "train", "epoch": 6}
{"loss": 0.6521238088607788, "acc": 0.632354736328125, "length": 3.2265625, "mode": "test", "epoch": 6}
{"loss": 0.6602979898452759, "acc": 0.625585675239563, "length": 3.5547404289245605, "mode": "train", "epoch": 7}
{"loss": 0.6463999152183533, "acc": 0.6340819597244263, "length": 3.91796875, "mode": "test", "epoch": 7}
{"loss": 0.6588017344474792, "acc": 0.625440239906311, "length": 3.5622482299804688, "mode": "train", "epoch": 8}
{"loss": 0.6466728448867798, "acc": 0.630859375, "length": 3.98828125, "mode": "test", "epoch": 8}
{"loss": 0.6567726135253906, "acc": 0.6259478330612183, "length": 3.5588250160217285, "mode": "train", "epoch": 9}
{"loss": 0.6452335715293884, "acc": 0.6324219107627869, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.6531198620796204, "acc": 0.6273275017738342, "length": 3.5865087509155273, "mode": "train", "epoch": 10}
{"loss": 0.643791675567627, "acc": 0.6322265267372131, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.6472114324569702, "acc": 0.6319363117218018, "length": 3.6694843769073486, "mode": "train", "epoch": 11}
{"loss": 0.6436329483985901, "acc": 0.6328125, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.6399350166320801, "acc": 0.6369379162788391, "length": 3.7455132007598877, "mode": "train", "epoch": 12}
{"loss": 0.6288979053497314, "acc": 0.6468749642372131, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.6284798383712769, "acc": 0.6472395062446594, "length": 3.804816961288452, "mode": "train", "epoch": 13}
{"loss": 0.6169142127037048, "acc": 0.6605468392372131, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.6118659377098083, "acc": 0.6665254831314087, "length": 3.870657205581665, "mode": "train", "epoch": 14}
{"loss": 0.6046351194381714, "acc": 0.681640625, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.5917308926582336, "acc": 0.6942890286445618, "length": 3.9091098308563232, "mode": "train", "epoch": 15}
{"loss": 0.5921028256416321, "acc": 0.6990233063697815, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.5693393349647522, "acc": 0.7147493958473206, "length": 3.9343042373657227, "mode": "train", "epoch": 16}
{"loss": 0.585870623588562, "acc": 0.701171875, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.5514057278633118, "acc": 0.723699688911438, "length": 3.957601547241211, "mode": "train", "epoch": 17}
{"loss": 0.602977991104126, "acc": 0.6935548186302185, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.544297993183136, "acc": 0.7239875197410583, "length": 3.9645984172821045, "mode": "train", "epoch": 18}
{"loss": 0.5950464606285095, "acc": 0.703125, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.5317475199699402, "acc": 0.7297894954681396, "length": 3.9687795639038086, "mode": "train", "epoch": 19}
{"loss": 0.5927695631980896, "acc": 0.7087889909744263, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.5229406952857971, "acc": 0.7434314489364624, "length": 3.9729084968566895, "mode": "train", "epoch": 20}
{"loss": 0.5883233547210693, "acc": 0.7087891101837158, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.5140894651412964, "acc": 0.7475526332855225, "length": 3.9749162197113037, "mode": "train", "epoch": 21}
{"loss": 0.574231743812561, "acc": 0.734375, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.5052906274795532, "acc": 0.759346604347229, "length": 3.9758973121643066, "mode": "train", "epoch": 22}
{"loss": 0.5701948404312134, "acc": 0.7322266101837158, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.4948999583721161, "acc": 0.7672867178916931, "length": 3.981454610824585, "mode": "train", "epoch": 23}
{"loss": 0.5560683012008667, "acc": 0.7435546517372131, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.48561570048332214, "acc": 0.7705399990081787, "length": 3.9837498664855957, "mode": "train", "epoch": 24}
{"loss": 0.5472483038902283, "acc": 0.7521483898162842, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.47635796666145325, "acc": 0.776314377784729, "length": 3.9882471561431885, "mode": "train", "epoch": 25}
{"loss": 0.5370483994483948, "acc": 0.7542968392372131, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.46924883127212524, "acc": 0.78242027759552, "length": 3.99039363861084, "mode": "train", "epoch": 26}
{"loss": 0.5278977751731873, "acc": 0.7576172351837158, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.4603893756866455, "acc": 0.784453272819519, "length": 3.9920127391815186, "mode": "train", "epoch": 27}
{"loss": 0.5199585556983948, "acc": 0.7658202648162842, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.45729854702949524, "acc": 0.7867399454116821, "length": 3.991239547729492, "mode": "train", "epoch": 28}
{"loss": 0.5226593613624573, "acc": 0.7496094107627869, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.45139995217323303, "acc": 0.7878154516220093, "length": 3.9935479164123535, "mode": "train", "epoch": 29}
{"loss": 0.5165364742279053, "acc": 0.7511718273162842, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.44685977697372437, "acc": 0.7870361804962158, "length": 3.9946396350860596, "mode": "train", "epoch": 30}
{"loss": 0.5191736817359924, "acc": 0.7560548186302185, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.4427332878112793, "acc": 0.7899000644683838, "length": 3.994178056716919, "mode": "train", "epoch": 31}
{"loss": 0.5112612247467041, "acc": 0.7601562142372131, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.4417366087436676, "acc": 0.7894900441169739, "length": 3.99440336227417, "mode": "train", "epoch": 32}
{"loss": 0.51444411277771, "acc": 0.7521485090255737, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.4381760060787201, "acc": 0.7916870713233948, "length": 3.9945497512817383, "mode": "train", "epoch": 33}
{"loss": 0.5088028907775879, "acc": 0.7552734017372131, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.436678022146225, "acc": 0.793086588382721, "length": 3.995518922805786, "mode": "train", "epoch": 34}
{"loss": 0.5040885210037231, "acc": 0.7638671398162842, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.43302327394485474, "acc": 0.7917168736457825, "length": 3.996663808822632, "mode": "train", "epoch": 35}
{"loss": 0.5131734609603882, "acc": 0.7587890625, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.4318121373653412, "acc": 0.7916024923324585, "length": 3.9962775707244873, "mode": "train", "epoch": 36}
{"loss": 0.5040050745010376, "acc": 0.7546873688697815, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.4289899468421936, "acc": 0.7935580015182495, "length": 3.9968857765197754, "mode": "train", "epoch": 37}
{"loss": 0.5003474950790405, "acc": 0.7609376311302185, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.42513370513916016, "acc": 0.7947717308998108, "length": 3.9967520236968994, "mode": "train", "epoch": 38}
{"loss": 0.49685925245285034, "acc": 0.7640624642372131, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.42550474405288696, "acc": 0.7968915104866028, "length": 3.9964828491210938, "mode": "train", "epoch": 39}
{"loss": 0.49551087617874146, "acc": 0.7595703601837158, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.42245498299598694, "acc": 0.7976515889167786, "length": 3.997248411178589, "mode": "train", "epoch": 40}
{"loss": 0.4875929355621338, "acc": 0.7601562142372131, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.42268213629722595, "acc": 0.7961084246635437, "length": 3.9978513717651367, "mode": "train", "epoch": 41}
{"loss": 0.4848242998123169, "acc": 0.7654297351837158, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.4190158247947693, "acc": 0.7994376420974731, "length": 3.9974794387817383, "mode": "train", "epoch": 42}
{"loss": 0.48614099621772766, "acc": 0.7652344107627869, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.4159369170665741, "acc": 0.8036399483680725, "length": 3.99796462059021, "mode": "train", "epoch": 43}
{"loss": 0.48413607478141785, "acc": 0.7628905773162842, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.4119763970375061, "acc": 0.8016838431358337, "length": 3.9977591037750244, "mode": "train", "epoch": 44}
{"loss": 0.4843127429485321, "acc": 0.7624998688697815, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.41154924035072327, "acc": 0.8029031753540039, "length": 3.9982106685638428, "mode": "train", "epoch": 45}
{"loss": 0.4822230339050293, "acc": 0.7652344107627869, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.4086276590824127, "acc": 0.8041495084762573, "length": 3.9971764087677, "mode": "train", "epoch": 46}
{"loss": 0.4876692593097687, "acc": 0.7605469226837158, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.4077652394771576, "acc": 0.8053761124610901, "length": 3.998342990875244, "mode": "train", "epoch": 47}
{"loss": 0.47777676582336426, "acc": 0.7707031965255737, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.40509745478630066, "acc": 0.8057821393013, "length": 3.998518466949463, "mode": "train", "epoch": 48}
{"loss": 0.47928914427757263, "acc": 0.7660155892372131, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.4027251601219177, "acc": 0.8090619444847107, "length": 3.9986486434936523, "mode": "train", "epoch": 49}
{"loss": 0.47740134596824646, "acc": 0.7718749642372131, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.4001321792602539, "acc": 0.8121759295463562, "length": 3.999016046524048, "mode": "train", "epoch": 50}
{"loss": 0.4752548933029175, "acc": 0.7705078125, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.39661142230033875, "acc": 0.8158044219017029, "length": 3.9991652965545654, "mode": "train", "epoch": 51}
{"loss": 0.47779858112335205, "acc": 0.7646484375, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.3929380476474762, "acc": 0.8125825524330139, "length": 3.9991586208343506, "mode": "train", "epoch": 52}
{"loss": 0.4773300886154175, "acc": 0.7761718034744263, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.39215677976608276, "acc": 0.8144789934158325, "length": 3.998957872390747, "mode": "train", "epoch": 53}
{"loss": 0.4764259159564972, "acc": 0.7740234732627869, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.38735267519950867, "acc": 0.8196583390235901, "length": 3.9991698265075684, "mode": "train", "epoch": 54}
{"loss": 0.47796696424484253, "acc": 0.7716796398162842, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.3878278434276581, "acc": 0.8208507895469666, "length": 3.9987199306488037, "mode": "train", "epoch": 55}
{"loss": 0.4772225618362427, "acc": 0.7765624523162842, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.3850770890712738, "acc": 0.8217301368713379, "length": 3.9992401599884033, "mode": "train", "epoch": 56}
{"loss": 0.4760335683822632, "acc": 0.7845702767372131, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.3817179501056671, "acc": 0.8273101449012756, "length": 3.999173641204834, "mode": "train", "epoch": 57}
{"loss": 0.4754941761493683, "acc": 0.7814453840255737, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.378888338804245, "acc": 0.8253009915351868, "length": 3.999347448348999, "mode": "train", "epoch": 58}
{"loss": 0.4637314975261688, "acc": 0.7843749523162842, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3772316873073578, "acc": 0.8249617218971252, "length": 3.999624729156494, "mode": "train", "epoch": 59}
{"loss": 0.4700280427932739, "acc": 0.7841797471046448, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.3782464563846588, "acc": 0.8273190855979919, "length": 3.9995763301849365, "mode": "train", "epoch": 60}
{"loss": 0.4641113579273224, "acc": 0.7873048186302185, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.6355357766151428
